proto/iotics/api/meta.proto

```diff
@@ -24,11 +24,21 @@ service MetaAPI {
 
   // SparqlQuery performs a SPARQL 1.1 query and returns one or more results, each as a sequence of chunks. Note that:
   // - Chunks for a particular result will arrive in-order though they might be interleaved with chunks from other
-  //   results (when performing a non-local query). See scope parameter in SparqlQueryRequest.
+  //   results (when performing a non-local query). See scope parameter in SparqlQueryRequest;
   // - The call will only complete once the (specified or host default) request timeout has been reached. The client can
   //   choose to end the stream early once they have received enough results. (E.g. in the case of Scope.LOCAL this
   //   would be after the one and only sequence of chunks has been received.)
   rpc SparqlQuery(SparqlQueryRequest) returns (stream SparqlQueryResponse) {}
+
+  // SparqlUpdate performs a SPARQL 1.1 update with write access to a local custom metadata graph and returns a
```

Comment:
Since the graph must be explicitly specified - I think maybe this should be slightly different - otherwise it sounds like, by default, triples will automatically be written into this graph.

How about e.g.:
> SparqlUpdate performs a SPARQL 1.1 update. When performing an update, the update query must contain a reference to one of the following graph IRIs:
>
> 1) http://data.iotics.com/graph#custom-public (AKA custom public graph) - All metadata written to this graph will be visible during SPARQL queries both with local & global scope (and thus the Iotics network).

Reply:
and ditch the SPARQL snippet?